lib/android/src/main/java/net/skyscanner/backpack/reactnative/calendar/CalendarViewManager.kt

```diff
@@ -59,8 +59,8 @@ class CalendarViewManager : ViewGroupManager<RNCalendarView>() {
   }
 
   @ReactProp(name = "selectionType")
-  fun setSelectionType(view: RNCalendarView, selectionType: String) {
-    view.state.selectionType = when (selectionType) {
+  fun setSelectionType(view: RNCalendarView, selectionType: ReadableMap) {
+    view.state.selectionType = when (selectionType.getString("type")) {
```

Comment:
Here's an example of the code from our website:
selectionType={rangeSelection}

It seems that it doesn't use any keys here. Please make sure it works okay.

Reply:
`rangeSelection` is an object containing `type: "range"`